packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -329,11 +330,14 @@ class Neo4jGraphQL {
 
     private generateExecutableSchema(): Promise<GraphQLSchema> {
         return new Promise((resolve) => {
-            const document = this.getDocument(this.typeDefs);
+            const initialDocument = this.getDocument(this.typeDefs);
 
             const validationConfig = this.parseStartupValidationConfig();
 
-            validateDocument({ document, validationConfig });
+            validateDocument({ document: initialDocument, validationConfig });
+
+            const { document, typesExcludedFromGeneration } = makeSchemaToAugment(initialDocument);
+            const { jwtPayload } = typesExcludedFromGeneration;
```

Comment:
Not important: Personally, I would have preferred having this `exclusion` logic be a responsibility of `makeAugmentedSchema`, but I believe we can address that during the refactor of it!